aiohttp/test_utils.py

```diff
@@ -413,7 +413,7 @@ def unittest_run_loop(func, *args, **kwargs):
     """
 
     @functools.wraps(func, *args, **kwargs)
-    def new_func(self):
+    def new_func(self, *args, **kwargs):
```

Comment:
When you use other decorators, for example `@patch` with `@unittest_run_loop` then:
1. `unittest_run_loop()` transform your test-function to `new_func(self)`
2. `@patch` (or other decorator) will applied for `new_func()` which can't take argument from the patch